### ISSUE TITLE
Payment Options

### ISSUE THREAD

**Issue by thubamamba** [scottyzen/woonuxt#259]
Firstly,this is not so much an issue. 

I just stumbled upon this project and my biggest question is regarding payment gateways, can we add our own payment gateways? For instance, we don't use Stripe but use Paystack and others.

**Comment by grandant** [comment]
I don't think you can use other payment gateways out of the box. The author of the project has mentioned that WooNuxt currently supports Stripe, PayPal, and Cash on Delivery, with plans to expand this in the future. That said, it’s possible to add your own integrations, but it might require some effort, as WooNuxt appears to be somewhat tightly coupled with the existing payment options on the WP backend.

As for the checkout, I think the author has done a great job to provide a smooth and fully integrated user experience. While you could customize WooNuxt to disrupt this flow, it feels counterproductive to use a modern Nuxt app only to redirect to WordPress at the final step.
